pkg/cmd/controller/controller_workflow_integration_test.go

```diff
@@ -46,9 +47,9 @@ func TestSequentialWorkflow(t *testing.T) {
 	stagingRepoName := "environment-staging"
 	prodRepoName := "environment-production"
 
-	fakeRepo := gits.NewFakeRepository(testOrgName, testRepoName)
-	stagingRepo := gits.NewFakeRepository(testOrgName, stagingRepoName)
-	prodRepo := gits.NewFakeRepository(testOrgName, prodRepoName)
+	fakeRepo, _ := gits.NewFakeRepository(testOrgName, testRepoName, nil, nil)
```

Comment:
Would it not be better to keep `NewFakeRepository` as is and introduce something like `NewLocal(Test)Repository` instead. Spares the retrofitting with `nil`s and is more accurate name wise.